player/lua/defaults.lua

```diff
@@ -1,5 +1,5 @@
 -- Compatibility shim for lua 5.2/5.3
-unpack = unpack or table.unpack
+local unpack = unpack or table.unpack -- luacheck: globals unpack table.unpack
```

Comment:
personally I would keep this.

Reply:
Restored, but for the record, not a big fun of it.